thingsboard_gateway/connectors/opcua/opcua_connector.py

```diff
@@ -118,6 +118,7 @@ def close(self):
         self.__log.info("Stopping OPC-UA Connector")
 
         asyncio.run_coroutine_threadsafe(self.__cancel_all_tasks(), self.__loop)
+        asyncio.run(self.__disconnect())
```

Comment:
We need to run it before cancel_all_tasks and using the same run_coroutine_threadsafe function, so the code will look like:
```python
asyncio.run_coroutine_threadsafe(self.__disconnect(), self.__loop)
asyncio.run_coroutine_threadsafe(self.__cancel_all_tasks(), self.__loop)
```